pkg/3scale/amp/component/ampimages.go

```diff
@@ -575,7 +575,7 @@ func (ampImages *AmpImages) buildQuayServiceAccount() *v1.ServiceAccount {
 		},
 		ImagePullSecrets: []v1.LocalObjectReference{
 			v1.LocalObjectReference{
-				Name: "quay-auth"}}}
+				Name: "threescale-registry-auth"}}}
```

Comment:
function still called `buildQuayServiceAccount`, kind of weird. isn't it?

Reply:
fixed